### PR TITLE
Initial version of sheet for 'The Troubleshooters' by Helmgast

### PR DESCRIPTION
**Pull request by Fragpuss**
Initial version of sheet for 'The Troubleshooters' by Helmgast

Brand new sheet for a new game and my first attempt at creating a sheet, since no one else has so far. I've done my best to follow the (legacy) standards. Uses multiple tabs to try to capture the flavour of the 'Emergency Passport' character sheet published by Helmgast. I have contacted the creators and have permission to use the imagery with an appropriate note at the bottom of the sheet.
## Roll20 Requests

Comments are very helpful for reviewing the code changes. Please answer the relevant questions below in your comment.

- [ ] Does the pull request title have the sheet name(s)? Include each sheet name.
- [ ] Is this a bug fix?
- [ ] Does this add functional enhancements (new features or extending existing features) ?
- [ ] Does this add or change functional aesthetics (such as layout or color scheme) ? 
- [ ] If changing or removing attributes, what steps have you taken, if any, to preserve player data ?
- [ ] If this is a new sheet, did you follow [Building Character Sheets standards](https://wiki.roll20.net/Building_Character_Sheets#Roll20_Character_Sheets_Repository) ?

If you do not know English. Please leave a comment in your native language.

### CODE REVIEW

**Review by Fragpuss — COMMENTED** [review]
Updated the sheet.json file to include missing comma

File: The Troubleshooters/sheet.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+	"html": "TheTroubleshooters.html",
+	"css": "TheTroubleshooters.css",
+	"authors": "Richard M.",
+	"roll20userid": "3094711",
+	"preview": "TheTroubleshooters.png",
+	"instructions": "RM June 2021 - This sheet is for the Swedish RPG The Troubleshooters (2021) published by Helmgast (https://helmgast.se/en/troubleshooters/). The coat of arms image from the passport cover and The Troubleshooters name and logo are (c) Helmgast and are used here with their kind permission. \n - When learning languages or developing attributes, use the checkboxes to record experience spent. \n - Condition descriptions are revealed by checking the boxes. \n When adding weapons on the Combat sheet, selecting the applicable skill will reveal the relevant roll buttons. \n This sheet can be edited and improved as long as the copyright statement is retained/updated for any relevant imagery or other content used.",
```
Comment (Anduh) [inline The Troubleshooters/sheet.json:7]:
```suggestion
	"instructions": "RM June 2021 - This sheet is for the Swedish RPG [The Troubleshooters](https://helmgast.se/en/troubleshooters/) (2021) published by Helmgast. The coat of arms image from the passport cover and The Troubleshooters name and logo are (c) Helmgast and are used here with their kind permission. \n - When learning languages or developing attributes, use the checkboxes to record experience spent. \n - Condition descriptions are revealed by checking the boxes. \n When adding weapons on the Combat sheet, selecting the applicable skill will reveal the relevant roll buttons. \n This sheet can be edited and improved as long as the copyright statement is retained/updated for any relevant imagery or other content used.",
```
You can use markdown to make hyperlinks and to do some formatting/styling for the description.

I recommend looking into CSE, as there are new features it enables, and actually makes sheet creation simpler, as you don't need `sheet-` in from of every class. [Datalists](https://wiki.roll20.net/CSE#Datalist) is among the simplest features it enables, giving you the best of a text input & a select drop-down.

**Review by Anduh — CHANGES_REQUESTED** [review]
I strongly recommend you change your sheetworkers to be inside a single element.

Even if it might work with having two blocks at times, it will cause troubles/bugs.

**Comment by Fragpuss** [comment]
> 
> 
> I strongly recommend you change your sheetworkers to be inside a single element.
> 
> Even if it might work with having two blocks at times, it will cause troubles/bugs.

Thanks, I've changed it appropriately, I think. Am I correct that now it's committed I don't need to create a new pull request?

**Comment by Anduh** [comment]
> Thanks, I've changed it appropriately, I think. Am I correct that now it's committed I don't need to create a new pull request?

Yes, that's how it works. Any updates to the barnch you've made a pull request from will automatically be updated to the PR. Ofc now that's it's merged, you'll need to make a new PR, from a new branch, if you want to do any future updates to the sheet.

**Comment by Fragpuss** [comment]
> 
> 
> > Thanks, I've changed it appropriately, I think. Am I correct that now it's committed I don't need to create a new pull request?
> 
> Yes, that's how it works. Any updates to the barnch you've made a pull request from will automatically be updated to the PR. Ofc now that's it's merged, you'll need to make a new PR, from a new branch, if you want to do any future updates to the sheet.

Aah - a new branch - as it happens, I need to make a couple of changes - some of the formatting is a bit off - it looked fine in the sandbox, and after a bit of experimentation, I think I need to change the legacy setting in the json. Also, one of the images didn't display, are the filenames in the src case-sensitive?

Thanks again for your advice.

**Comment by Anduh** [comment]
>  are the filenames in the src case-sensitive?
Likely.

**Pull request merged**